### PR TITLE
More chains added

### PR DESCRIPTION
**Pull request by stackman27**

### CODE REVIEW

File: selectors.yml [hunk 1]
```diff
@@ -155,6 +158,9 @@ selectors:
   432201:
     selector: 1458281248224512906 
     name: "avalanche-subnet-dexalot-testnet"
+  717160:
+    selector: 4418231248214522936 
+    name: "ethereum-testnet-polygon-validium-1"
```
Comment (andrevmatos) [inline selectors.yml:163]: There's no `ethereum-testnet` called `polygon`; if `polygon-validium` is the `1`st instance of this rollup, then it must be attached to some ethereum-testnet, and that needs to be specified here

File: selectors.yml [hunk 2]
```diff
@@ -60,6 +60,9 @@ selectors:
   1112:
     selector: 9284632837123596123
     name: "wemix-testnet" 
+  12325:
+    selector: 3486622437121596122 
+    name: "arbitrum-orbit-l3x-testnet"
```
[the remark this answers is not in the document]
Comment (stackman27) [reply selectors.yml:65]: l3x is a layer 3 built using arbitrum orbit. how about `arbitrum-testnet-orbit-l3x-testnet`?

File: selectors.yml [hunk 3]
```diff
@@ -167,6 +173,9 @@ selectors:
   810181: 
     selector: 5837261596322416298
     name: "zklink_nova-testnet" 
+  31415926:
```
[the remark this answers is not in the document]
Comment (stackman27) [reply selectors.yml:176]: added mainnet https://github.com/smartcontractkit/chain-selectors/pull/41/commits/292842310ccbf8bbb9d6d5ed3baf8b2d25c63ea3

File: selectors.yml (hunk 1 above)
Comment (stackman27) [reply selectors.yml]: it is a sepolia testnet chain at a high level so changed it to `ethereum-testnet....` 
https://github.com/smartcontractkit/chain-selectors/pull/41/commits/292842310ccbf8bbb9d6d5ed3baf8b2d25c63ea3

[the remark this answers is not in the document]
Comment (stackman27) [reply selectors.yml]: changed https://github.com/smartcontractkit/chain-selectors/pull/41/commits/292842310ccbf8bbb9d6d5ed3baf8b2d25c63ea3

File: selectors.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mateusz-sekara) [reply selectors.yml:65]: I mean if it's layer 3 then it should end up with rollup instance, right? 
https://github.com/smartcontractkit/chain-selectors?tab=readme-ov-file#adding-new-chains

[the remark this answers is not in the document]
Comment (stackman27) [reply selectors.yml:65]: how about `ethereum-testnet-sepolia-arbitrum-l3x-1`?

[the remark this answers is not in the document]
Comment (mateusz-sekara) [reply selectors.yml:65]: Okay, let's have that one `ethereum-testnet-sepolia-arbitrum-l3x-1`, please make sure similar naming is reflected for the mainnet one

**Review by mateusz-sekara — DISMISSED** [review]

**Pull request merged**